generated/nidmm/enums.py

```diff
@@ -180,76 +165,76 @@ class DigitsResolution(Enum):
 class Function(Enum):
     DC_VOLTS = 1
     '''
-    All devices supported.
+    DC Voltage
     '''
     AC_VOLTS = 2
     '''
-    All devices supported.
+    AC Voltage
     '''
     DC_CURRENT = 3
     '''
-    All devices supported.
+    DC Current
     '''
     AC_CURRENT = 4
     '''
-    All devices supported.
+    AC Current
     '''
-    _2_WIRE_RESISTANCE = 5
+    _2_WIRE_RES = 5
     '''
-    All devices supported.
+    2-Wire Resistance
     '''
-    _4_WIRE_RESISTANCE = 101
+    _4_WIRE_RES = 101
     '''
-    NI 4065, and NI 4070/4071/4072 supported.
+    4-Wire Resistance
     '''
-    FREQUENCY = 104
+    FREQ = 104
     '''
-    NI 4070/4071/4072 supported.
+    Frequency
     '''
     PERIOD = 105
     '''
-    NI 4070/4071/4072 supported.
+    Period
     '''
     TEMPERATURE = 108
     '''
     NI 4065, and NI 4070/4071/4072 supported.
     '''
     AC_VOLTS_DC_COUPLED = 1001
     '''
-    NI 4070/4071/4072 supported.
+    AC Voltage with DC Coupling
     '''
     DIODE = 1002
     '''
-    All devices supported.
+    Diode
     '''
     WAVEFORM_VOLTAGE = 1003
     '''
-    NI 4070/4071/4072 supported.
```

Comment:
Look at that, a bunch of outdated documentation talking about EOLed devices and not the new flagships 408x.
(mentioning specific models in API documentation is always a maintenance nightmare)

Reply:
Internal CAR 675265 filed